### PR TITLE
Api helpers

### PR DESCRIPTION
**Pull request by Lordnickim**
- Added two apiHelpers for getProductStylesById and getRelatedProductsById 
- Included two routes for the new apihelpers
- Updated getProductById by having 'req.params.id' to be 'req.params.product_id' to resolve undefine issue.

### CODE REVIEW

**Review by Lordnickim — COMMENTED** [review]
Darn thought if i created a newbranch wouldnt have to worry about other commits from the card branch :/

File: apiHelpers/atelier.js [hunk 1]
```diff
@@ -1 +1,3 @@
-const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-la/'
\ No newline at end of file
+const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-la/'
```
Comment (erikeh) [inline apiHelpers/atelier.js]: hr-lax

**Review by erikeh — APPROVED** [review]

File: apiHelpers/atelier.js (hunk 1 above)
Comment (Lordnickim) [reply apiHelpers/atelier.js]: ahhhhhhhh

File: apiHelpers/atelier.js [hunk 2]
```diff
@@ -21,10 +21,41 @@ module.exports = {
...
+      console.log(res.data)
+      return res.data
+    })
+    .catch(err => {
+      console.log(err)
+      return err;
+    })
+  },
+
+  getStlyesById: function(product_id) {
```
Comment (raphaelspies) [inline apiHelpers/atelier.js]: fix typo in getStylesById ... and check any calls to this function

File: apiHelpers/atelier.js (hunk 1 above)
Comment (Lordnickim) [reply apiHelpers/atelier.js]: wait lols howd you find that line? looks like its changed in the commit thou? 😖

Comment (erikeh) [reply apiHelpers/atelier.js]: ahh it looks like i was looking at an outdated commit. I think next time I'm just gonna look at the 'Files changed' section to see the LATEST!

Comment (Lordnickim) [reply apiHelpers/atelier.js]: Lools I see it interesting can see each commit changes separately

File: server/routes/products.js [hunk 3]
```diff
@@ -31,12 +31,28 @@ products
 // return all styles available for the given product
 products
   .route('/:product_id/styles')
-  .get()
+  .get((req, res) => {
+    return API.getStlyesById(req.params.product_id)
```
Comment (raphaelspies) [inline server/routes/products.js]: this line references misspelled getStyleById -- fix typo

File: client/src/components/RelatedContainer.jsx [hunk 4]
```diff
@@ -9,23 +9,47 @@ export default function RelatedContainer () {
...
   //Method will be need to break up more below
-  const getProductInfo = () => {
-    axios.get('/product/productid')
-      .then((rawData) => {
-        setProductInfo([rawData.data])
-      })
-      .catch((err) => {
-        console.log('Error getting Product...', err)
-      })
+  async function getProductsInfo(Id) {
```
Comment (raphaelspies) [inline client/src/components/RelatedContainer.jsx]: Does this function work? If not, please save for a separate pull request.

**Review by raphaelspies — CHANGES_REQUESTED** [review]

**Review by erikeh — APPROVED** [review]

**Comment by Lordnickim** [comment]
@raphaelspies Changes should be good to go. Let me know so then I can start merging and havin clean branch bases XD

**Review by raphaelspies — APPROVED** [review]
LGTM!

**Pull request merged**